nominees/berkely-air-sums.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "Berkely Air Sums",
+    "description": "The Stove Use Monitoring System (SUMS) provides insights into cookstove usage patterns, number of meals cooked, and time of use by recording stove temperature changes. Data sampled by SUMS are uploaded to a computer via a data cable and further analyzed using proprietary algorithms developed to quantify cookstove usage in households.",
+    "website": "https://berkeleyair.com/",
+    "license": [
+      {
+        "spdx": "",
+        "licenseURL": ""
+      }
+    ],
+    "SDGs": [
+      {
+        "SDGNumber": 3,
+        "evidenceURL": "http://berkeleyair.com/about-us/mission/"
```

Comment:
Instead of having all evidenceURLs point to the project's website, [here](https://cookstoves.lbl.gov/research/sums/) is an external link providing evidence for SDG 3. @ijeomaemeruwa kindly update this evidenceURL to use the above URL. Thanks.